lte/gateway/python/magma/pipelined/rpc_servicer.py

```diff
@@ -987,7 +987,7 @@ def _ng_tunnel_update(self, pdr_entry: PDRRuleEntry, subscriber_id: str) -> bool
             pdr_entry.pdr_state,
             pdr_entry.precedence,
             pdr_entry.local_f_teid,
-            pdr_entry.far_action.o_teid,
+            pdr_entry.o_teid,
```

Comment:
are the changes in this file (and I assume related `session_state_manager_util.py)` covered by a test?

Reply:
Yes,  it's functionality verification done.

Reply:
I do not understand what you mean. I do not see a new test for these changes.

Reply:
Only Variable name and place was changed. Remining are same as functionality. We have already verified the functionality with simulator.